incubator/virtualcluster/README.md

```diff
@@ -35,36 +35,78 @@ Kubecon EU 2020 talk (~25 mins) | WG meeting demo (~50 mins)
-rejects a newly created tenant Pod if its `nodename` has been set in the spec.
+## FAQ
 
-- It is recommended to increase the tenant master node controller `--node-monitor-grace-period` parameter to a larger value
-( >60 seconds, done in the sample clusterversion [yaml](config/sampleswithspec/clusterversion_v1_nodeport.yaml) already).
-The syncer controller does not update the node lease objects in tenant master,
-hence the default grace period is too small.
+### Q: What is the difference between VirtualCluster and multi-cluster solution?
 
-- Coredns is not tenant-aware. Hence, tenant should install coredns in tenant master if DNS is required.
-The DNS service should be created in `kube-system` namespace using name `kube-dns`. The syncer controller can then
-recognize the DNS service cluster IP in super master and inject it into Pod spec `dnsConfig`.
+One of the primary design goals of VirtualCluster is to improve the overall resource utilization
+in super cluster by allowing multiple tenants to share the node resource in an isolated manner. 
+A multi-cluster solution can achieve the same isolation goal but the overall resource utilization
+is going to be suboptimal.
 
-- VirtualCluster supports tenant DNS service using a customized coredns build. See this [document](./doc/tenant-dns.md) for details.
+### Q: Can the tenant master run its own scheduler?
 
-- VirtualCluster fully support tenant service account.
+VirtualCluster was primarily designed for serverless use cases where users normally do not have
+scheduling preferences. Using the super cluster scheduler can much easily
+achieve good overall resource utilization. For these reasons, 
+VirtualCluster does not support tenant scheduler. It is technically possible
+to support tenant scheduler by exposing some of the super cluster nodes directly in
+tenant master. Those nodes have to be dedicated to the tenant to avoid any scheduling
+conflicts. This type of tenant should be exceptional.
 
-- VirtualCluster does not support tenant PersistentVolumes. All PVs and Storageclasses are provided by the super master.
+### Q: What is the difference between Syncer and Virtual Kubelet? 
 
-- It is recommended that tenant master and super master should use the same Kubernetes version to avoid
-incompatible API behaviors. The syncer controller and vn-agent are built using Kubernetes 1.16 APIs, hence
-higher Kubernetes versions are not supported at this moment.
+They have similarities. In some sense, the syncer controller can be viewed as the replacement of a virtual
+kubelet in cases where the resource provider of the virtual kubelet is a Kubernetes cluster. The syncer certainly does more
+though. It maintains the one to one mapping between a virtual node in tenant master and a real node
+in super cluster. It preserves the Kubernetes API compatibility as completely as possible. It provides fair queuing to
+mitigate tenant interference. It is a more native implementation for tenant abstraction.
```

Comment:
```suggestion
in the super cluster. It preserves the Kubernetes API compatibility as closely as possible. Additionally it provides fair queuing to
mitigate tenant interference. It is a more native implementation for tenant abstraction.
```

Reply:
What do you think about instead of `interference` we used `contention`?

Also, what do you mean by "It is a more native implementation for tenant abstraction."?